h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
The tests don't actually call `format()` though, they call `toFuzzyString()` which (sometimes) gets `dayAndMonth()` or `dayAndMonthAndYear()` (via `BREAKPOINTS` and `getBreakpoint()`) and those two both call `format()`. We'd have to add the optional `Intl` argument to `toFuzzyString()`, `dayAndMonth()`, `dayAndMonthAndYear()` and `format()` and pass it down. Or did you mean that we should add some direct unit tests for `format()`?